From: =?utf-8?q?Jonas_=C3=85dahl?= <jadahl@gmail.com>
Date: Tue, 9 Sep 2025 23:56:18 +0200
Subject: Use 'dbus' debug topic for D-Bus name changes

Lets make it a META_DEBUG_DBUS debug log entry instead. It's not useful
to have in the journal, and it interferes with tests trying to assert
that expected warnings are logged.

Bug: https://gitlab.gnome.org/GNOME/mutter/-/issues/4310
Forwarded: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/4648
---
 src/backends/meta-a11y-manager.c         | 4 ++--
 src/backends/meta-dbus-session-manager.c | 2 +-
 src/backends/meta-input-mapper.c         | 4 ++--
 src/core/meta-service-channel.c          | 4 ++--
 4 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/backends/meta-a11y-manager.c b/src/backends/meta-a11y-manager.c
index c30f7c3..d8f02fc 100644
--- a/src/backends/meta-a11y-manager.c
+++ b/src/backends/meta-a11y-manager.c
@@ -344,7 +344,7 @@ on_name_acquired (GDBusConnection *connection,
                   const char      *name,
                   gpointer         user_data)
 {
-  g_debug ("Acquired name %s", name);
+  meta_topic (META_DEBUG_DBUS, "Acquired name %s", name);
 }
 
 static void
@@ -352,7 +352,7 @@ on_name_lost (GDBusConnection *connection,
               const char      *name,
               gpointer         user_data)
 {
-  g_debug ("Lost or failed to acquire name %s", name);
+  meta_topic (META_DEBUG_DBUS, "Lost or failed to acquire name %s", name);
 }
 
 static void
diff --git a/src/backends/meta-dbus-session-manager.c b/src/backends/meta-dbus-session-manager.c
index db2728f..3944bca 100644
--- a/src/backends/meta-dbus-session-manager.c
+++ b/src/backends/meta-dbus-session-manager.c
@@ -165,7 +165,7 @@ on_name_lost (GDBusConnection *connection,
   MetaDbusSessionManagerPrivate *priv =
     meta_dbus_session_manager_get_instance_private (session_manager);
 
-  g_warning ("Lost or failed to acquire name %s", name);
+  meta_topic (META_DEBUG_DBUS, "Lost or failed to acquire name %s", name);
 
   priv->is_enabled = FALSE;
 
diff --git a/src/backends/meta-input-mapper.c b/src/backends/meta-input-mapper.c
index badcf67..a0d123e 100644
--- a/src/backends/meta-input-mapper.c
+++ b/src/backends/meta-input-mapper.c
@@ -887,7 +887,7 @@ on_name_acquired (GDBusConnection *connection,
                   const char      *name,
                   gpointer         user_data)
 {
-  g_info ("Acquired name %s", name);
+  meta_topic (META_DEBUG_DBUS, "Acquired name %s", name);
 }
 
 static void
@@ -895,7 +895,7 @@ on_name_lost (GDBusConnection *connection,
               const char      *name,
               gpointer         user_data)
 {
-  g_info ("Lost or failed to acquire name %s", name);
+  meta_topic (META_DEBUG_DBUS, "Lost or failed to acquire name %s", name);
 }
 
 static void
diff --git a/src/core/meta-service-channel.c b/src/core/meta-service-channel.c
index 363120e..37f08a7 100644
--- a/src/core/meta-service-channel.c
+++ b/src/core/meta-service-channel.c
@@ -398,7 +398,7 @@ on_name_acquired (GDBusConnection *connection,
                   const char      *name,
                   gpointer         user_data)
 {
-  g_info ("Acquired name %s", name);
+  meta_topic (META_DEBUG_DBUS, "Acquired name %s", name);
 }
 
 static void
@@ -406,7 +406,7 @@ on_name_lost (GDBusConnection *connection,
               const char      *name,
               gpointer         user_data)
 {
-  g_warning ("Lost or failed to acquire name %s", name);
+  meta_topic (META_DEBUG_DBUS, "Lost or failed to acquire name %s", name);
 }
 
 static void
